_posts/2016-04-30-akka-basics.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+name: Akka-basics
+title: 'Akka basics'
+date: 2016-04-30 00:20:00 +00:00
+author: Felipe Fernández
+image:
+    src: /assets/img/custom/blog/akka.jpg
+    attribution:
+      text: Akka_20120331 by Helena Whitlock, SGU
+      href: https://www.flickr.com/photos/geologicalsurveyofsweden/8703286181
+tags:
+- akka
+- OOP
+- functional programming
+
+---
+
+As [the previous post](http://codurance.com/2016/04/28/async-systems-with-sync-clients) explained our system is based on Akka. Before going into more detail about our solution, I'd like to explain what is Akka about and why is so awesome.
```

Comment:
Might be better if you close the sentence as:
what Akka is about and why it is so awesome.